bats/fb-test-puppet.bats

```diff
@@ -79,12 +79,12 @@ fi
 }
 
 @test "assign puppet class to host" {
-  id=$(hammer --csv puppet-class list --search 'name = ntp' | tail -n1 | cut -d, -f1)
+  id=$(hammer --csv puppet-class list --search 'name = motd' | tail -n1 | cut -d, -f1)
   pc_ids=$(hammer host update --help | awk '/class-ids/ {print $1}')
   hammer host update $pc_ids $id --name $(hostname -f)
 }
 
 @test "apply class with puppet agent" {
   puppet agent -v -o --no-daemonize
-  grep -i puppet /etc/ntp.conf
+  grep -i "File managed with puppet" /etc/motd
```

Comment:
As suggested in https://github.com/theforeman/puppet-motd/pull/1#discussion_r412734304

```suggestion
  grep -i 'property of the Foreman project' /etc/motd
```